waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -809,10 +819,10 @@
         ;; delay iff the log-bucket-url setting was given the scheduler config.
         log-bucket-sync-secs (if log-bucket-url (:log-bucket-sync-secs context) 0)
         total-sigkill-delay-secs (+ pod-sigkill-delay-secs log-bucket-sync-secs)
-        has-reverse-proxy? (get env "GRPC_TRANSCODER")
+        has-reverse-proxy? (some #(= (:name %) "GRPC_TRANSCODER")  service-description)
```

Comment:
Please pass the string `GRPC_TRANSCODER` as a configuration defined in the context argument of this function.